test/powershell/engine/Basic/Encoding.Tests.ps1

```diff
@@ -0,0 +1,74 @@
+Describe "File encoding tests" -tag CI {
+
+    Context "ParameterType for parameter 'Encoding' should be 'Encoding'" {
+        BeforeAll {
+            $testCases = get-command -module Microsoft.PowerShell.* | 
+                Where-Object { $_.Parameters -and $_.Parameters['Encoding'] } |
+                ForEach-Object { @{ Command = $_ } }
+        }
+        It "Encoding parameter of command '<Command>' is type 'Encoding'" -testcase $testCases {
+            param ( $command )
+            $command.Parameters['Encoding'].ParameterType.FullName | Should Be "System.Text.Encoding"
```

Comment:
Should be `BeExactly`?

Reply:
it could be - `BeExactly` is a case-sensitive string compare, but the case of the response is not the point, only that it is the proper fullname.
I'm in the file so I'll update it, but we should be cautious about being overly pedantic.